addons/udes_permissions/views/user_template_view.xml

```diff
@@ -0,0 +1,99 @@
+<odoo>
+    <data>
+        <!-- Form view -->
+        <record id="user_template_form" model="ir.ui.view">
+            <field name="name">user.template.form</field>
+            <field name="model">user.template</field>
+            <field name="arch" type="xml">
+                <form string="User Template">
+                    <sheet>
+                        <!-- Here we have a set of smart buttons that use the 'recursive_child_*' columns
+                        in combination with actions defined on user.template to display the relevant information -->
+                        <div name="button_box" class="oe_button_box">
+                            <button name="action_view_child_templates" type="object" class="oe_stat_button" icon="fa-sitemap">
+                                <field name="family_descendant_count" widget="statinfo" string="Child Templates" />
+                            </button>
+                            <button name="action_view_child_groups" type="object" class="oe_stat_button" icon="fa-users">
+                                <field name="family_group_count" widget="statinfo" string="Implied Groups" />
```

Comment:
(Just a comment to remind myself)

Not 100% sure on using `Implied`. Obviously it is implied for the template, but it isn't necessarily implied for the user that the template is applied to. Super low priority though and I can't think of an alternative atm anyway. :)

Reply:
Let's leave it as if... if you're managing user templates then you should be knowing this. And at the moment the parent templates inherit all groups the child has, so should be fine. :)